__tests__/programs_spec.js

```diff
@@ -18,6 +22,55 @@ beforeAll((async function () {
 
 describe('Programs Page Test Suite => ', function () {
 
+    it('All Elements Exists', async function () {
+        await page.goto(testconfig.resources.programsUrl, {waitUntil: 'networkidle2'});
+
+        // check header title
+        expect(await page.$(selectors.programs.title)).toBeDefined();
+        expect(await page.$eval(selectors.programs.title, s => s.innerText.trim()))
+            .toBe(texts.programs.title);
+
+        // check logo
+        expect(await page.$eval(selectors.programs.logo, s => s.innerText.trim()))
+            .toBe(texts.programs.logo);
+
+        // check vertical menu list
+        expect(await page.$$eval(selectors.programs.sideBar, (ss) => {
+            return ss.map(s => s.innerText.trim())
```

Comment:
Please write in the new way all this functions, where function return map should be two map instead of return. 

For example:

expect(await page.$$eval(selectors.main.horizonIconRows, ss => ss.map(s => s.innerText.trim())))
            .toEqual(texts.main.horizonIconRows);